builder/bundles.go

```diff
@@ -524,7 +631,14 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 		return err
 	}
 
-	return writeBundleInfoPretty(bundle, filepath.Join(metaPath, bundle.Name))
+	err = writeBundleInfoPretty(bundle, filepath.Join(metaPath, bundle.Name))
+	if err != nil {
+		return err
+	}
+
+	close(rpmCh)
+	wg.Wait()
+	return nil
```

Comment:
Move `close(rpmCh)` and `wg.Wait()` above `writeBundleInfoPretty` and keep the original return line.